### ISSUE TITLE
Mobile twitter is broken since 84.0.4147.113

### ISSUE THREAD

**Issue by Michalko589** [bromite/bromite#664]
### Bromite version

Version: 84.0.4147.113
Arch: arm64
Android version: 9.0
Device model: Galaxy S8

### Is this bug about the SystemWebView?
No

### Is the bug reproducible with latest version?

Yes

### Can the bug be reproduced with corresponding Chromium version?

No

### Is the bug a crash?

No

### Describe the bug

After the last update (84.0.4147.113), Twitter Mobile is no longer working. The error screen is looking the same as in [this screenshot](https://i.redd.it/xfcoywjaysw11.jpg) I found on the Internet, just without the bottom blue bar. Tried reinstalling Bromite, didn't help. Chromium 84.0.4147.113 is working just fine and so is Bromite 84.0.4147.106.

### Steps to reproduce the bug

Steps to reproduce the bug:
1. Go to mobile.twitter.com
2. Log in
3. "Something went wrong" appears instead of your feed

### Expected behavior

Twitter feed loading.

**Comment by Michalko589** [comment]
Oops, just noticed that in the screenshot I provided

**Comment by csagan5** [comment]
Using remote devices you can see that the error is:
```
Access to script at 'https://abs.twimg.com/responsive-web/web_legacy/i18n/en.9f143124.js' from origin 'https://mobile.twitter.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

So this seems due to CORS RFC1918 which is currently enabled.

**Comment by gut5** [comment]
Same issue here, switching off Secure DNS has fixed it for now but obviously this is not a desirable solution.

**Comment by Michalko589** [comment]
@gut5 I don't use Secure DNS, yet it is still broken.

**Comment by SV84** [comment]
Don't using Twitter, but have same problem on Reddit.

After log on, just show me 'Go back' page, using  last version.

**Comment by fitittome** [comment]
This seems relevant. 
https://www.chromium.org/Home/chromium-security/extension-content-script-fetches

**Comment by MariojGG** [comment]
Same problem. And not only in twitter, reddit and instagram too.

**Comment by elisonniven** [comment]
Instagram and Twitter work but Reddit doesn't work.

**Comment by csagan5** [comment]
Fixed in `84.0.4147.119`.
